tlsr9/crypto/mbedtls/internal/ecp_alt_b9x_backend.c

```diff
@@ -45,9 +45,16 @@
 #include <mbedtls/ecp.h>
 #include <mbedtls/error.h>
 #include <multithread.h>
+#if CONFIG_SOC_RISCV_TELINK_B91 || CONFIG_SOC_RISCV_TELINK_B92
 #include <pke.h>
+#elif CONFIG_SOC_RISCV_TELINK_B95
+#include <lib/include/pke/pke.h>
```

Comment:
same here
clear for all changes include paths to use same includes and avoid lots of #if